lib/cell.rb

```diff
@@ -0,0 +1,48 @@
+class Cell
+ attr_reader  :coordinate,
+              :ship
+
+  def initialize(coordinate)
+    @coordinate = coordinate
+    @ship = nil
+    @empty = true
+    @fired_upon = false
```

Comment:
The fired_upon was stumping me! After seeing your code, I was like   ;__;  I couldn't believe how silly I felt. :)

Reply:
Don't feel silly...I hate writing instance variables that closely resemble method names, but when the pry interaction requests a `?` that is significant, so this is the work around.  I guess the only fix is to make the instance variable past tense to suggest that it is a boolean, but I think I just made that up and it's not a steadfast truth or norm in programming.